### PR TITLE
- add a make file command for publishing a docker image

### PR DESCRIPTION
**Pull request by oibe**
- documentation

### CODE REVIEW

File: jaeger-crossdock/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Dockerized permutations testing for Jaeger
+
+This repository implments docker testing for jaeger-client-java.  To be more specific Jaeger
```
Comment (yurishkuro) [inline jaeger-crossdock/README.md]: typo: implements

Comment (yurishkuro) [inline jaeger-crossdock/README.md]: "docket testing" doesn't make sense - be "more specific" to from start

File: jaeger-crossdock/README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+# Usage #
+
+## Run Crossdock tests ##
+
+In order to run crossdock tests running `make crossdock-fresh` in the root directory of this
+project is sufficient.
+
+# Contributors #
+
+To push a new crossdock image run `make upload-image`.
```
Comment (yurishkuro) [inline jaeger-crossdock/README.md]: why is this under contributors?

Comment (oibe) [inline jaeger-crossdock/README.md]: I want some generic section for people working on this.

Comment (oibe) [inline jaeger-crossdock/README.md]: Its not for users.

Comment (yurishkuro) [inline jaeger-crossdock/README.md]: "Contributors" section is customary used to list contributors to the project, not instructions for contributors (CONTRIBUTING.md is used for the latter).

I don't like this new make command to begin with, and we certainly shouldn't put it in the readme. Uploading images to dockerhub is done by the CI builds, there's no reason to do that manually.

**Pull request closed without merging**